ownCloud/Client/Actions/Actions+Extensions/DeleteAction.swift

```diff
@@ -64,13 +64,13 @@ class DeleteAction : Action {
 						if error != nil {
```

Comment:
@pablocarmu just a minor complaint, may be extracting the code operating on an `item` within the loop into separately defined lambda would make the code more readable.